grib/src/test/java/ucar/nc2/grib/coord/TestDiscontiguousInterval.java

```diff
@@ -0,0 +1,50 @@
+package ucar.nc2.grib.coord;
+
+import org.junit.Assert;
+import org.junit.Test;
+import ucar.ma2.Array;
+import ucar.nc2.*;
+
+import java.io.IOException;
+
+public class TestDiscontiguousInterval {
+
+  @Test
+  public void testTimeCoordinate1D_isMonotonicallyIncreasing() throws IOException {
+    String testfile = "../grib/src/test/data/GFS_Global_onedeg_20220627_0000.precipOnly.grib2";
```

Comment:
Is it possible to make a smaller test file? I think 28 MB is a bit on the large size. Otherwise we could use an existing one in the `thredds-test-data` repo

Reply:
Switched to a much smaller test dataset with latest commit. So will mark resolved.